src/instance_normalization.js

```diff
@@ -31,10 +31,10 @@ export function instanceNormalization(
   const shape = [1, null, 1, 1];
   const reduceOptions = {axes: [2, 3], keepDimensions: true};
   const mean = reduceMean(input, reduceOptions);
-  output = sub(input, reshape(mean, shape));
+  output = sub(input, mean);
   const variance = reduceMean(pow(output, new Scalar(2)), reduceOptions);
```

Comment:
Why not match the pseudo code?
```js
const variance = reduceMean(pow(sub(input, mean), new Scalar(2)), reduceOptions);
```

Reply:
Last I referred to implementation of **batchNormalization**, missed modified **sub** and **add** computing of this implementation. Now it matches, after this fixing.

Reply:
I mean match it exactly by combining line 34 into line 35.